eventsource/stores/dynamodbstore/store.go

```diff
@@ -43,11 +44,7 @@ func (store *store) LoadByAggregate(ctx context.Context, aggregateID string) (re
 
 	output, err := store.db.QueryWithContext(ctx, &input)
 	if err != nil {
-		log.
-			WithField("input", input).
-			WithField("error", err).
-			Error("Couldn't scan pages")
-		err = errors.Wrap(err, "couldn't scan pages")
+		err = fmt.Errorf("couldn't scan pages (input=%+v): %w", input, err)
```

Comment:
Could you remove the log and change from wrros.Wrap to fmt.Errorf for the other methods in the file also?

Reply:
Done.